Tests/LibWeb/Crash/Editing/execCommand-comment-boundary.html

```
@@ -0,0 +1,5 @@
+<div contenteditable><!----></div>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)